app/templates/power-ups/topics.html

```diff
@@ -40,40 +85,28 @@
   });
 }
   </pre>
-  <p>Popups can be nested, which will automatically provide a back button in the corner of the popup. You can also request that
-    search be allowed on the popup. This will provide native web client functionality for performing full text searches of
-    your supplied items.
-  </p>
+  <p>Popups can be nested, which will automatically provide a back button in the corner of the popup. You can also request that search be allowed on the popup. This will provide native web client functionality for performing full text searches of your supplied items.</p>
   <p>Alternatively, popups can be created containing an iframe by supplying a title, URL, and height instead of a set of items.</p>
+  <hr/>
 
-  <h3>Attachment</h3>
-  <p>Attachments can be created with the <strong>t.attach</strong> method, but they can only be removed by the user using the
-    standard web experience in the attachments section, or by clicking in the upper right hand corner of any attachment-section
-    claiming an attachment.</p>
+  <h2 id="attachments">Attachments</h2>
+  <p>Attachments can be created with the <strong>t.attach</strong> method, but they can only be removed by the user using the standard web experience in the attachments section, or by clicking in the upper right hand corner of an attachment-section claiming an attachment.</p>
```

Comment:
Might be worth mentioning that the success of this call depends on the member on whose behalf the Power-Up is making it. So a member interacting with your Power-Up who is not a member of the board, this will fail.

Power-Ups can check this in advance by looking at the context. The context has a `permissions` property, with sub-properties of `card`, `board`, and `organization`. If you look at the context and look at `permissions.card` and see that it is `'write'` then you can generally expect this call will succeed. Power-Ups may want to alter their behavior (for example not even showing a button at all) based on the permissions of the current user.